### PR TITLE
Remove unsupported versions

### LINKED ISSUE #124: Drop support for django 1.x and python 2.7

### PR DESCRIPTION
**Pull request by ulgens**
Fixes #124

### CODE REVIEW

**Comment by ulgens** [comment]
@alicertel Do you hava any idea about why this PR didn't trigger a Travis build?

**Comment by alicertel** [comment]
Hi @ulgens looks like travis run the tests https://travis-ci.org/github/Hipo/drf-extra-fields but some how it doesn't reflected to the PR. I will look into it.

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Django >= 1.4
-djangorestframework >= 3.0.1
\ No newline at end of file
+Django >= 2.2
+djangorestframework >= 3.9.2
```
[the remark this answers is not in the document]
Comment (ulgens) [reply requirements.txt:2]: 3.9.2 is the earliest version that supports Django 2.2. Any earlier version will create resolution issues. https://www.django-rest-framework.org/community/release-notes/#392

[the remark this answers is not in the document]
Comment (alicertel) [reply requirements.txt:2]: Makes sense, thank you @ulgens  Sorry for being late.

**Pull request merged**